corehq/apps/user_importer/tests/test_importer.py

```diff
@@ -1767,6 +1808,46 @@ def test_upper_case_email(self):
         user_history = UserHistory.objects.get()
         self.assertNotIn('email', user_history.changes)
 
+    def test_user_data(self):
+        self.setup_users()
```

Comment:
off-topic, but any idea why `setup_users` isn't part of `setUp`?

Reply:
It looks like one of the test needed to patch something in user creation code. I refactored it so that it's part of setUp now.

https://github.com/dimagi/commcare-hq/pull/33930/commits/e95ba67e7c2a24e61a782a546fac8ef367b0de3a